pkg/trace/api/pipeline_stats.go

```diff
@@ -68,6 +70,7 @@ func pipelineStatsErrorHandler(err error) http.Handler {
 // newPipelineStatsProxy creates an http.ReverseProxy which forwards requests to the pipeline stats intake.
 // The tags will be added as a header to all proxied requests.
 func newPipelineStatsProxy(conf *config.AgentConfig, urls []*url.URL, apiKeys []string, tags string) *httputil.ReverseProxy {
+	log.Debug("[pipeline_stats] Creating reverse proxy")
```

Comment:
What do you think about also adding a debug log when payloads are received?

Reply:
done